trulens_eval/trulens_eval/tests/answer_relevance_smoke_tests.ipynb

```diff
@@ -0,0 +1,239 @@
+    "In many ways, feedbacks can be thought of as LLM apps themselves. Given text, they return some result. Thinking in this way, we can use TruLens to evaluate and track our feedback quality. We can even do this for different models (e.g. gpt-3.5 and gpt-4) or prompting schemes (such as chain-of-thought reasoning).\n",
+    "\n",
+    "This notebook follows an evaluation of a set of test cases. You are encouraged to run this on your own and even expand the test cases to evaluate performance on test cases applicable to your scenario or domain."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import relevance feedback function\n",
+    "from trulens_eval.feedback import GroundTruthAgreement, OpenAI\n",
+    "from trulens_eval import TruBasicApp, Feedback, Tru, Select\n",
+    "from test_cases import answer_relevance_golden_set"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "os.environ[\"OPENAI_API_KEY\"] = \"...\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "turbo = OpenAI(model_engine=\"gpt-3.5-turbo\")\n",
+    "# Define your feedback functions\n",
+    "def wrapped_relevance_turbo(input, output):\n",
+    "    return turbo.relevance(input, output)\n",
+    "\n",
+    "# Define your feedback functions\n",
+    "def wrapped_relevance_with_cot_turbo(input, output):\n",
+    "    return turbo.relevance_with_cot_reasons(input, output)\n",
+    "\n",
+    "gpt4 = OpenAI(model_engine=\"gpt-4\")\n",
+    "# Define your feedback functions\n",
+    "def wrapped_relevance_gpt4(input, output):\n",
+    "    return gpt4.relevance(input, output)\n",
+    "\n",
+    "# Define your feedback functions\n",
+    "def wrapped_relevance_with_cot_gpt4(input, output):\n",
+    "    return gpt4.relevance_with_cot_reasons(input, output)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "✅ In Relevance Smoke Test, input prompt will be set to *.__record__.calls[0].args.args[0] .\n",
+      "✅ In Relevance Smoke Test, input response will be set to *.__record__.calls[0].args.args[1] .\n",
+      "✅ In Relevance Smoke Test, input score will be set to *.__record__.main_output or `Select.RecordOutput` .\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Create a Feedback object using the numeric_difference method of the ground_truth object\n",
+    "ground_truth = GroundTruthAgreement(answer_relevance_golden_set)\n",
+    "# Call the numeric_difference method with app and record\n",
+    "f_groundtruth = Feedback(ground_truth.numeric_difference, name = \"Relevance Smoke Test\").on(Select.Record.calls[0].args.args[0]).on(Select.Record.calls[0].args.args[1]).on_output()"
```

Comment:
Can you explain 2 pieces of info in each notebook
*  the golden set is populated with a lookup table on both prompt/response exact match
* The numeric difference is the L1 distance between expected and actual score